bundles/net.tourbook/icon-factory/common.svg

```diff
@@ -7510,5 +7510,14 @@
+       class="cls-1"
+       id="path6"
+       style="stroke-width:0.04622141"
+       inkscape:export-filename="C:\Users\frederic\git\MT\mytourbook\bundles\net.tourbook\icons\hard-drive.png"
```

Comment:
The hard drive image is a static image which is put into common.svg and then exported. All svg files MUST contain only self painted images to have no legal trouble.

![hard-drive-image](https://user-images.githubusercontent.com/1283445/87217918-af0dbd00-c34e-11ea-9dbe-9ecc134ff921.png)

Reply:
I have obtained this SVG via _Wikimedia Commons, the free media repository_

Here is the [link](https://commons.wikimedia.org/wiki/File:Hard-drive.svg) where you can see that it is legal to use this SVG.

But if you prefer, I can draw one myself, let me know